app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer_test.go

```diff
@@ -616,10 +579,8 @@ func verifyRemoveClustersResult(lbc *Syncer, expectedClusters, expectedIGlinks [
 		return fmt.Errorf("unexpected IG links on firewall rule, expected: %v, got: %v", expectedFWIGLinks, fw.IGLinks)
 	}
 	// Verify that the load balancer is spread to both clusters.
-	if err := verifyClusters(lbc, expectedClusters); err != nil {
-		return err
-	}
-	return nil
+	err := verifyClusters(lbc, expectedClusters)
```

Comment:
I personally prefer code 1 over code 2.
code 1:
```go
if err := doStep1(); err != nil {
  return err
}
if err := doStep2(); err != nil {
  return err
}
return nil
```
code 2:
```go
if err := doStep1(); err != nil {
  return err
}
return doStep2()
```
While code 2 is less lines, code 1 allows for easy extension (can add/remove/reorder steps easily).

If you do want to switch this to code 2, you can combine both the lines to return verifyClusters directly

Reply:
This was a lint error when it was "code 1", which was surprising!

Reply:
I just tried lint on HEAD and didnt get any error?